lte/gateway/c/oai/lib/pcef/CMakeLists.txt

```diff
@@ -33,5 +33,6 @@ target_include_directories(LIB_PCEF PUBLIC
   ${MAGMA_LIB_DIR}/async_grpc
   ${MAGMA_LIB_DIR}/config
   ${MAGMA_LIB_DIR}/service_registry
+  ${CMAKE_CURRENT_SOURCE_DIR}/../../tasks/sgw/
```

Comment:
This is to include spgw_types.h. Please place the spgw_types.h in c/oai/include directory